.github/workflows/pr_reviews.yml

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Adding Reviewers
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  add_reviewers:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Add Reviewers
+      run: |
+        user_json=$(gh api users/${{ github.event.review.user.login }})
+        user_name=$(echo "$user_json" | jq -r '.name')
+        user_email=$(echo "$user_json" | jq -r '.email')
+        if [[ "${{ github.event.pull_request.body }}" =~ ^.*Reviewers:\ .*${user_name}.*$ ]]; then
+            echo "Reviewer already added: ${user_name} <${user_email}>"
+        elif [[ "${{ github.event.pull_request.body }}" =~ ^.*Reviewers:\ .*$ ]]; then
+          pr_body="${{ github.event.pull_request.body }}, ${user_name} <${user_email}>"
+          gh pr edit ${{ github.event.pull_request.number }} --body "${pr_body}"
+          echo "Added reviewer: ${user_name} <${user_email}>"
+        else
+          pr_body="${{ github.event.pull_request.body }}
+
+        Reviewers: ${user_name} <${user_email}>"
```

Comment:
indentation seems off here

Reply:
It's intentional. The "Reviews: " part would have an indentation on the PR as it literally sets the `pr_body` var as the new PR body. In other words the bash string made out of it will contain the indentation (hence the empty line as well).